packages/upload-client/README.md

```diff
@@ -253,7 +284,7 @@ function add(
 ): Promise<CID>
 ```
 
-Store a CAR file to the service.
+Store a CAR file to the service. Returns the CID of the CAR file stored.
```

Comment:
Is the returned CID the CID of the input car file? Or might it be a different CID due to sharding? If the latter, this might be more accurate (but idk?)
```suggestion
Store a CAR file to the service. Returns the CID of the sharded CAR file.
```

Reply:
`Store.add` does not shard, it stores whatever CAR file you pass it. This is part of the reason for the `uploadCAR` function.